COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@jagajaga,

> I think that discussion about exactly this PR is over after @kernc's message?

I believe @eduarrrd is the only one qualified to judge whether this patch is his private property or that of his employer, so I'd rather not presume to tell him what to do.

What I *can* say is that I, personally, will not merge a patch that modifies the top-level `COPYING` file without getting explicit consent from a representative part of the NixOS community first. If this means that this patch cannot be merged, then that's unfortunate but it's the way it is. Other collaborators might feel differently about this and they might be willing to merge the patch as-is. If that is so, then more power to them! It's just that I won't.